pkg/docker/containers-view.jsx

```diff
@@ -129,6 +129,29 @@ var ContainerDetails = React.createClass({
     }
 });
 
+var ContainerProblems = React.createClass({
+    onItemClick: function (event) {
+        cockpit.jump(event.currentTarget.dataset.url, cockpit.transport.host);
+    },
+
+    render: function () {
```

Comment:
I was going to complain about the non-ES6 style methods, but the whole file is like that, so let's clean this up separately.